### PR TITLE
Map changes to block changes in 21w11a

### PR DESCRIPTION
**Pull request by haykam821**

### CODE REVIEW

File: mappings/net/minecraft/Oxidizable.mapping [hunk 1]
```diff
@@ -0,0 +1,18 @@
+CLASS net/minecraft/class_5955 net/minecraft/Oxidizable
```
Comment (liach) [inline mappings/net/minecraft/Oxidizable.mapping]: Need a package, like it was in `block` before. See https://github.com/FabricMC/yarn/commit/696ae0e310862ea032428dd106448b3f749fa0be#diff-630aea1ba64f09736359a6e96df90b95d6bc66f158378d3bf9197ad2b0ba0e7f

File: mappings/net/minecraft/item/AxeItem.mapping [hunk 2]
```diff
@@ -9,3 +9,9 @@ CLASS net/minecraft/class_1743 net/minecraft/item/AxeItem
 		ARG 4 settings
 	METHOD method_20269 (Lnet/minecraft/class_1838;Lnet/minecraft/class_1657;)V
 		ARG 1 p
+	METHOD method_34716 getStrippedState (Lnet/minecraft/class_2680;)Ljava/util/Optional;
```
Comment (liach) [inline mappings/net/minecraft/item/AxeItem.mapping]: Does all stripping go through this? I suggest
```suggestion
	METHOD method_34716 tryStrip (Lnet/minecraft/class_2680;)Ljava/util/Optional;
```

Comment (haykam821) [reply mappings/net/minecraft/item/AxeItem.mapping]: It doesn't actually strip any blocks, just gets the stripped state with the same axis property using the stripped block map.

Comment (liach) [reply mappings/net/minecraft/item/AxeItem.mapping]: :+1: Right. Please mark this one as resolved.

File: mappings/net/minecraft/Oxidizable.mapping (hunk 1 above)
Comment (haykam821) [reply mappings/net/minecraft/Oxidizable.mapping]: Fixed in 8a5e2db4b072358419e72883d7dfd195f64bf50f

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/block/Block.mapping [hunk 3]
```diff
@@ -39,6 +39,13 @@ CLASS net/minecraft/class_2248 net/minecraft/block/Block
 		ARG 3 pos
 		ARG 4 state
 	METHOD method_33615 getShapesForStates (Ljava/util/function/Function;)Lcom/google/common/collect/ImmutableMap;
+	METHOD method_34724 copyProperty (Lnet/minecraft/class_2680;Lnet/minecraft/class_2680;Lnet/minecraft/class_2769;)Lnet/minecraft/class_2680;
+		ARG 0 source
+		ARG 1 target
+		ARG 2 property
+	METHOD method_34725 getStateWithProperties (Lnet/minecraft/class_2680;)Lnet/minecraft/class_2680;
+		COMMENT Gets a block state with all properties that both this block and the source block state have.
```
Comment (Juuxel) [inline mappings/net/minecraft/block/Block.mapping:47]:
```suggestion
		COMMENT Gets a state of this block with all properties that both this block and the source block state have.
```
A small clarification.

**Review by Juuxel — APPROVED** [review]

**Review by jaskarth — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Pull request merged**